docs/sql-directory.md

```diff
@@ -24,7 +24,7 @@ All update files are found in the $\data\sql\updates\ directory.
 
 These files contain all the updates committed.
 
-Files from this directory are automatically imported with the [Automatic Database Updater](database-installation#automatic-database-updater-default-recommended).
+Files from this directory are automatically imported with the [Automatic Database Updater](database-installation#automatic-database-updater).
```

Comment:
Los archivos de este directorio se importan automáticamente con el [Actualizador automático de bases de datos]

for Files from this directory are automatically imported with the [Automatic Database Updater]